src/controller/eventController.js

```diff
@@ -0,0 +1,57 @@
+
+var events=[];
+  app1.controller("EventController", function($scope,$location,$http,$routeParams,
```

Comment:
dependency injection should be in array notation, for reference: https://github.com/practo/javascript/blob/master/guides/angular.md#dependencies